barcode/tests/__init__.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+from barcode import *
+from barcode_utils import generate_barcode_from_badge_num, get_badge_num_from_barcode
+
+
+@pytest.fixture
+def cfg(monkeypatch):
+    config['secret'] = dict()
+    config['secret']['barcode_key'] = "ABCDEF1234"
+    config['secret']['barcode_salt'] = 42
+    config['secret']['barcode_event_id'] = 0xFF
+
+
+def test_cfg_fixtures(cfg):
```

Comment:
yea, gotcha. makes sense.  actually is there anyplace where we validate config on startup, I could extend that to plugins and put these checks here? @EliAndrewC ?